dolphinscheduler-master/src/main/java/org/apache/dolphinscheduler/server/master/engine/command/handler/RunWorkflowCommandHandler.java

```diff
@@ -98,6 +98,7 @@ protected void assembleWorkflowExecutionGraph(final WorkflowExecuteContextBuilde
                             .builder()
                             .workflowExecutionGraph(workflowExecutionGraph)
                             .workflowDefinition(workflowExecuteContextBuilder.getWorkflowDefinition())
+                            .project(workflowExecuteContextBuilder.getProject())
```

Comment:
Also need to update `RecoverFailureTaskCommandHandler` and `WorkflowFailoverCommandHandler`.

Reply:
> Also need to update `RecoverFailureTaskCommandHandler` and `WorkflowFailoverCommandHandler`.

done